packages/react-storage/src/components/StorageBrowser/Views/Controls/Refresh.tsx

```diff
@@ -34,8 +34,8 @@ const RefreshButton = withBaseElementProps(Button, {
   variant: 'refresh',
 });
 
-export const RefreshControl: RefreshControl = () => (
-  <RefreshButton>
+export const RefreshControl: RefreshControl = ({ onClick, disabled }) => (
+  <RefreshButton onClick={onClick} {...(disabled ? { disabled: true } : {})}>
```

Comment:
Maybe missing some context here but think we can avoid the spread logic?
```suggestion
export const RefreshControl: RefreshControl = ({ onClick, disabled = false }) => (
  <RefreshButton onClick={onClick} disabled={disabled}>
```

Reply:
oh thanks Caleb, for some reason last time I tried it locally, I was getting an eslint error about the value needing to be omitted if it was boolean. I just tried your changes locally and I don't seem to get it anymore so maybe I just set up something incorrectly the first time 🤦